flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FileStoreSource.java

```diff
@@ -69,7 +92,10 @@ public FileStoreSource(
             boolean latestContinuous,
             @Nullable int[][] projectedFields,
             @Nullable Predicate partitionPredicate,
-            @Nullable Predicate fieldPredicate) {
+            @Nullable Predicate fieldPredicate,
+            @Nullable Long specifiedSnapshotId,
+            @Nullable
+                    Map<BinaryRowData, Map<Integer, List<DataFileMeta>>> specifiedManifestEntries) {
```

Comment:
What's the case that we will set those two fields with non-null values ?  I see all of them will be set to be `null` in the `buildFileStore`.

Reply:
> What's the case that we will set those two fields with non-null values ? I see all of them will be set to be `null` in the `buildFileStore`.

For `ALTER TABLE ... COMPACT`, the `buildFileStore` will be modified in the following PR